source/codegen/metadata/nifgen/attributes.py

```diff
@@ -339,6 +339,19 @@
         'resettable': False,
         'type': 'ViString'
     },
+    1150246: {
```

Comment:
Be careful while merging this change. #110 also adds both of these attributes. The second PR that goes in should not have these changes.

Reply:
the other pr is merged so that change is not showing anymore